ksql-engine/src/test/java/io/confluent/ksql/util/QueryLoggerUtilTest.java

```diff
@@ -0,0 +1,31 @@
+package io.confluent.ksql.util;
```

Comment:
Add license header. Same comment also for SerdeTestUtils, ProcessingLogMessageFactoryTest, and ProcessingLoggerUtilTest.